CRM/ManualDirectDebit/Upgrader.php

```diff
@@ -226,11 +250,23 @@ public function onEnable() {
   }
 
   public function uninstall() {
+    $this->deleteScheduledJob();
     $this->deletePaymentProcessor();
     $this->alterCustomValues('uninstall');
     $this->alterCustomGroups('uninstall');
     $this->deleteDirectDebitNavigationMenu();
-    $this->deleteMessageTemplates();
+  }
+
+  /**
+   * Deletes scheduled job created by the extension
+   *
+   * @throws \CiviCRM_API3_Exception
+   */
+  private function deleteScheduledJob() {
+    civicrm_api3('Job', 'get', [
```

Comment:
We also need to implement onEnable and onDisable events to handle what happens with the scheduled job. Basically, we need to set the `is_active` flag to true on enable, false on disable.